packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+# 51.8.0 - April 07, YYYY
+
+## Added
+
+#### salesforcedx-vscode-apex
+
+- Add progress reporting and cancellation to Apex test runs ([PR #3103](https://github.com/forcedotcom/salesforcedx-vscode/pull/3103))
+
+#### salesforcedx-sobjects-faux-generator
+
+- Add cancellation to SObject Refresh command ([PR #3116](https://github.com/forcedotcom/salesforcedx-vscode/pull/3116))
+
+## Fixed
+
+#### docs
+
+- Update Org Picker location in documentation ([PR #3099](https://github.com/forcedotcom/salesforcedx-vscode/pull/3099)) ([Issue #3095](https://github.com/forcedotcom/salesforcedx-vscode/issues/3095))
+
+- Update Apex status to reflect default usage in CLI ([PR #3119](https://github.com/forcedotcom/salesforcedx-vscode/pull/3119))
```

Comment:
@AnanyaJha are you porting this PR to the release branch ? I don't see it there yet, only in `develop`.

Reply:
Yessir port PR is here https://github.com/forcedotcom/salesforcedx-vscode/pull/3130

Reply:
Awesome 👍